src/Altinn.Apps/AppFrontend/react/shared/__tests__/utils/langugage.test.ts

```diff
@@ -101,4 +101,20 @@ describe('>>> src/Altinn.Apps/AppFrontend/react/shared/src/utils/language.ts', (
     textResource = mockTextResources.find((resource) => resource.id === 'mockId-1');
     expect(textResource.value).toEqual('Hello, cat!');
   });
+
+  it('+++ should replace multiple references to same value', () => {
+    mockTextResources = [
+      {
+        id: 'mockId',
+        value: 'This is a {0} apple. It will always be {0}. Yes, {0} is my favorite color.',
+        unparsedValue: 'This is a {0} apple. It will always be {0}. Yes, {0} is my favorite color.',
+        variables: [
+          { key: 'model.text.color', dataSource: 'dataModel.test' },
+        ],
+      },
+    ];
+    replaceTextResourceParams(mockTextResources, mockDataSources);
+    const textResource = mockTextResources.find((resource: ITextResource) => resource.id === 'mockId');
+    expect(textResource.value).toEqual('This is a yellow apple. It will always be yellow. Yes, yellow is my favorite color.');
```

Comment:
Would it not be easier to read the expected value if it was a string where `model.text.color` was inserted into the slots?
What if the mock changes, will all the tests have to be re-written?
example:
```JavaScript
expect(textResource.value)
.toEqual(`This is a ${mockValue}  apple. It will always be ${mockValue}. Yes, ${mockValue} is my favorite color.`);
```